pepdbagent/utils.py

```
@@ -93,3 +94,16 @@ def tuple_converter(value: Union[tuple, list, str, None]) -> tuple:
     return tuple(
         " ",
     )
+
+
+def convert_date_string_to_date(date_string: str) -> datetime.datetime:
+    """
+    Convert string into datetime format
+
+    :param date_str: date string in format [YYYY/MM/DD]. e.g. 2022/02/22
+    :return: datetime format
+    """
+    try:
+        return datetime.datetime.strptime(date_string, "%Y/%m/%d") + datetime.timedelta(days=1)
+    except Exception:
```

This will mask exceptions, don't do this.